packages/prisma/extensions/exclude-locked-users.ts

```diff
@@ -35,8 +43,10 @@ async function excludeLockedUsers(
   query: <T>(args: T) => Promise<unknown>
 ) {
   args.where = args.where || {};
+  const whereString = safeJSONStringify(args.where);
+  const shouldIncludeLocked = whereString.includes('"locked":false');
```

Comment:
Based on this var name, shouldn’t this be checking for locked: true?

Reply:
Actually we should only check for the property name on ensure is not undefined.
```suggestion
  const shouldIncludeLocked = whereString.includes('"locked":');
```